src/biome/text/modules/heads/classification/classification.py

```diff
@@ -39,8 +43,10 @@ def __init__(
         self._multilabel = multilabel
 
         # metrics and loss
+        if class_weights != None:
```

Comment:
It's more common to use `variable is not None`, since there is only one [instance of None](https://stackoverflow.com/questions/2209755/python-operation-vs-is-not)
```suggestion
        if class_weights is not None:
```